### PR TITLE
[docs]: Fix documentation for tabs components 

### LINKED ISSUE #11060: [docs]: Update the documentation for the Tabs Component
### Summary

**Formatting updates:**
- Replace  "`Fx`" and "FX" with "**fx**" wherever applicable. ( i.e. remove the backticks (``) make it lowercase and bold -> **fx**)
- For each h2 (created using ##), remove 24px padding-bottom. The `div` that wraps up the h2 headers and the section below the header should only have a "padding-top" of 24px and no "padding-bottom". 

**Content updates:**
- Replace the word "widget" with "component" wherever applicable. ( Please do not update the URLs )
- Layout Tabel
  - Update the title name from "Layout" to "Devices" [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices)
  - Update the name of the first column from  "Layout" to "Property"
  - Update the content in the "Description" and "Expected value" columns as per the given [reference](https://docs.tooljet.com/docs/widgets/text-input/#devices) ( Please just check the sentence structure from the given reference )
    - Add a full stop at the end of the sentence in the "Description" column.
- Add a divider before the "Styles" section [reference](https://docs.tooljet.com/docs/widgets/textarea#styles)

**The changes are to be implemented in two files -** 
 Next - ToolJet/docs/docs/widgets/tabs.md
 Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/tabs.md

### Issue Type

Documentation bug

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/widgets/tabs

### Steps to reproduce the issue

_No response_

### Additional Information

_No response_

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

### PR DESCRIPTION
**Pull request by yunus102**
[Click here to view the issue](https://github.com/ToolJet/ToolJet/issues/11060)

----

will close #11060

### CODE REVIEW

File: docs/docs/widgets/tabs.md [hunk 1]
```diff
@@ -103,30 +103,32 @@ Following actions of Tabs component can be controlled using the component specif
...
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Makes the component visible in desktop view. | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
+| Show on mobile  | Makes the component visible in mobile view.  | You can set it with the toggle button or dynamically configure the value by clicking on **fx** and entering a logical expression. |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
+
+***
```
Comment (RudraPratik30) [inline docs/docs/widgets/tabs.md]: Hey @yunus102, can you please use `----` instead of `***` for adding a divider?

**Review by RudraPratik30 — APPROVED** [review]
Hey @yunus102, thanks for contributing!
The changes look good to me and forwarding it to the team for final review.
You can expect your PR to be merged soon.

File: docs/docs/widgets/tabs.md (hunk 1 above)
Comment (yunus102) [reply docs/docs/widgets/tabs.md]: Sure, I’ll use ---- as the divider from now on. Thanks for letting me know!

**Comment by yunus102** [comment]
Hey @RudraPratik30, 
Thank you for the feedback and for moving it forward! Glad to hear the changes look good. I'll keep an eye out for the merge.

**Review by karan-rathod-316 — APPROVED** [review]
Looks good to me.

**Pull request merged**